DoSomething/quasar
# Queue abstract

#### What's this PR do?
Adds abstracted QuasarQueue processing class to prep for ingesting multiple different queue messages. It's designed so the `on_message` function will be over-ridden by child classes to perform whatever operations we need either per queue, or per queue type message.
#### Where should the reviewer start?
Added file below.
#### How should this be manually tested?
Checkout branch, and if you have appropriate Blink Queue credentials, setup test python class with single instance of QuasarQueue, then run queue.start_consume. You should see one queued message and its contents every half second.

#### Any background context you want to provide?
All of our queue logic has traditionally been in the customerio.py file, but it has tightly coupled logic to only handle certain queue types.
#### What are the relevant tickets?
